src/qt/overviewpage.cpp

```diff
@@ -124,8 +124,8 @@ OverviewPage::OverviewPage(const PlatformStyle *platformStyle, QWidget *parent)
 {
     ui->setupUi(this);
 
-    // use a SingleColorIcon for the "out of sync warning" icon
-    QIcon icon = platformStyle->SingleColorIcon(":/icons/warning");
+    // use red icon for the "out of sync warning" icon
+    QIcon icon(":/icons/outofsync");
```

Comment:
It's not in the header. Of course I could put it in the header, but that would make it a global function, which is not good, right?